addon/components/polaris-action-list/item.js

```diff
@@ -6,43 +6,115 @@ import layout from '../../templates/components/polaris-action-list/item';
 export default Component.extend({
   tagName: 'li',
 
+  attributeBindings: ['role', 'active:aria-selected'],
+
   layout,
 
   /**
-   * The item to display
-   *
-   * Supported properties:
-   *  - content
-   *  - url (not currently supported)
-   *  - destructive
-   *  - disabled
-   *  - icon
-   *  - image
-   *  - ellipsis (not currently supported)
-   *  - badge (not currently supported)
-   *  - onAction
-   *
-   * @property item
```

Comment:
Needed to remove this and pass individual properties through to support the new `role` property passed in from `polaris-action-list/section`